client/src/components/App/Content/InvestigationForm/TabManagement/ClinicalDetails/ClinicalDetails.tsx

```diff
@@ -133,13 +133,14 @@ const ClinicalDetails: React.FC = (): JSX.Element => {
                     </b>
                 </Typography>
                 <Autocomplete
-                    test-id='currentQuarantineAddress'
+                    test-id='currentQuarantineCity'
```

Comment:
Why not testId

Reply:
`testId` is a name of a paramter_, while `test-id` is the name of the actual HTML attribute.
Some components need to pass a parameter to sub-components and some don't.